flow/templates/base_status.jinja

```diff
@@ -1,14 +1,16 @@
 {# The following variables are available to all scripts. #}
+{% if overview or detailed %}
+{% set column_width_operation = [column_width_operation, 5] | max %}
```

Comment:
Is this necessary? There's already a default of 5 in the Python code now.

Reply:
You're right. Removed.